crates/nu-command/src/filesystem/ls.rs

```diff
@@ -59,7 +60,7 @@ impl Command for Ls {
             .switch("all", "Show hidden files", Some('a'))
             .switch(
                 "long",
-                "Get all available columns for each entry (slower; columns are platform-dependent)",
+                "Get almost all available columns for each entry (slower; columns are platform-dependent)",
```

Comment:
What is almost supposed to mean here? That kind of sounds like we're saying, we have all of them but we're not going to show you all of them.